### PR TITLE
Update gitlab-pipelines.yaml

### PR DESCRIPTION
**Pull request by ValberJunior**
Gitlab Pipelines Plugin

Hello, we are building our documentation in a section of our site, so I have changed the link to the plugin documentation to be directed to this new page.

new doc ➡️ https://platform.vee.codes/plugin/Gitlab%20Pipelines

Minor changes to the title and name of the plugin, we forgot to put the "Code" starting with a capital "C".

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ✔️ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/data/plugins/gitlab-pipelines.yaml [hunk 1]
```diff
@@ -13,4 +13,4 @@ tags:
   - gitlabCi
   - pipelines
   - jobs
-addedDate: '2023-09-27'
+addedDate: '2023-10-27'
```
Comment (Rugvip) [inline microsite/data/plugins/gitlab-pipelines.yaml:16]:
```suggestion
addedDate: '2023-09-27'
```

Keeping this as is since that's when it was added

**Review by Rugvip — APPROVED** [review]
Nice, thank you! 👍

**Comment by ValberJunior** [comment]
Hey, what's up, guys?

Does anyone know if there was an error in this pull request? 
I uploaded one for the same change to our other github plugin, and his change has already been published, but this one still has the same status.

If necessary, I can close this pull request and open a new one. :)

**Comment by awanlin** [comment]
Hi @ValberJunior, it's set to auto merge but look like the DCO check is stuck, let me see if I can get that resolved first

**Comment by freben** [comment]
Odd, it seems to be broken. Welp, the signoffs are in place, so I'll force merge anyway.

**Pull request merged**